webserver/jersey/src/main/java/io/helidon/webserver/jersey/JerseySupport.java

```diff
@@ -664,4 +681,19 @@ public void reload(ResourceConfig configuration) {
             throw new UnsupportedOperationException("Reloading is not supported in Helidon");
         }
     }
+
+    /**
+     * A low priority mapper to catch internal exceptions in Jersey and log them.
+     * Otherwise, these errors can get lost and never reported by Helidon. One example
+     * is a {@code ClassNotFoundException} exception in Jersey's code.
+     */
+    @Priority(Priorities.USER + 1000)
```

Comment:
A naïve comment: `USER` surprised me here, since this has nothing to do with the user, but maybe it's correct?

Reply:
Users should be able to override this one if they want to, so the priority of this one should be higher.